### PR TITLE
Re-adding Cat Scream (Normalized)

### PR DESCRIPTION
**Pull request by Kyogon**
## About The Pull Request

This PR is to re-add the removed Cat Scream after normalization edits I performed to make it less ear-blasting.

## How This Contributes To The Nova Sector Roleplay Experience

The cats have long awaited for the ability to scream like a cat once more. Now that the audio has been normalized, it will be on par with the other screams for loudness while portraying the scream that cats desire for RP. This in turn allows any felinid, cat, or cat-hybrid to stand out from normal humans like they did before!

## Proof of Testing
<details>

https://github.com/NovaSector/NovaSector/assets/30851775/05211151-c811-4c2d-93fb-e8324bc9fc23

 This video provides all the proof of the audio compared to other commonplace sounds like a door opening, taking an ID out of a PDA, and opening/closing a garbage bin.
</details>

## Changelog

:cl:
add: Re-Adds Cat Scream to the list of available screams to choose in character creation
/:cl:

### CODE REVIEW

File: modular_nova/modules/emotes/sound/voice/cat.ogg
Comment (GoldenAlpharex) [inline modular_nova/modules/emotes/sound/voice/cat.ogg:1]: Just rename this to `cat_scream.ogg` please, so it's easier to find.

File: modular_nova/modules/emotes/code/scream_datums.dm [hunk 1]
```diff
@@ -58,6 +58,11 @@ GLOBAL_LIST_EMPTY(scream_types)
 	male_screamsounds = list('modular_nova/modules/emotes/sound/voice/scream_lizard.ogg')
 	female_screamsounds = null
 
+/datum/scream_type/cat
+	name = "Cat Scream"
+	male_screamsounds = list('modular_nova/modules/emotes/sound/voice/cat.ogg')
```
Comment (GoldenAlpharex) [inline modular_nova/modules/emotes/code/scream_datums.dm:63]:
```suggestion
	male_screamsounds = list('modular_nova/modules/emotes/sound/voice/cat_scream.ogg')
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
🐱

File: modular_nova/modules/emotes/sound/voice/cat.ogg
Comment (Kyogon) [reply modular_nova/modules/emotes/sound/voice/cat.ogg:1]: Sure, will do, had it as cat to match the others but it would be better on this name scheme!

Comment (GoldenAlpharex) [inline modular_nova/modules/emotes/sound/voice/cat.ogg:1]: You forgot to remove this one.

Comment (Kyogon) [reply modular_nova/modules/emotes/sound/voice/cat.ogg:1]: Got it!

**Review by GoldenAlpharex — APPROVED** [review]
🐱

**Pull request merged**